package.json

```diff
@@ -20,8 +20,8 @@
     "url": "https://github.com/ssh-iitmandi/baat-cheet/issues"
   },
   "dependencies": {
+    "ejs": "^2.5.5",
     "express": "^4.14.0",
-    "socket.io": "^1.7.2",
-    "ejs": "^2.5.5"
+    "socket.io": "^2.3.0"
   }
-}
+}
```

Comment:
Do not update these dependencies, because at present heroku doesn't support latest versions, app crashes when we update dependencies. You can remove this file.